src/main/resources/templates/index.html

```diff
@@ -34,19 +34,21 @@ <h3>Select your departing city</h3> <br />
 
     <select name="seatClass" id="classSelect" required>
         <option value="">Select Seat Class</option>
-        <option value="firstClass" id="firstClassSeats">First Class $300, Seats Available: x</option>
-        <option value="businessClass" id="businessClassSeats">Business Class $200, Seats Available: x</option>
-        <option value="economyClass" id="economyClassSeats">Economy Class $100, Seats Available: x</option>
+        <option value="first" id="firstClassSeats">First Class $300, Seats Available: x</option>

```

Comment:
Adjusted the option values to match the ones used in the flight service. Added a small description to the priority boarding pricing.